rust/examples/classification-example/src/main.rs

```diff
@@ -83,6 +85,27 @@ fn sort_results(buffer: &[f32]) -> Vec<InferenceResult> {
     results
 }
 
+fn image_to_tensor(path: String, height: u32, width: u32) -> Vec<u8> {
```

Comment:
This needs some documentation explaining that the file at `path` is resized to `height x width` and then the pixel precision is converted to FP32.